python/py-hatch-fancy-pypi-readme/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-hatch-fancy-pypi-readme
+version             22.3.0
+revision            0
+categories-append   devel
+license             MIT
+supported_archs     noarch
+
+python.versions     37 38 39 310
+python.pep517       yes
+python.pep517_backend hatch
+
+maintainers         nomaintainer
+
+description         a Hatch metadata plugin for projects' PyPI landing page
+long_description    {*}${description}
+
+homepage            https://github.com/hynek/hatch-fancy-pypi-readme
+distname            hatch_fancy_pypi_readme-${version}
+
+checksums           rmd160  715d9e3a3300cb6c9f04d7c4b45479aa13410631 \
+                    sha256  7d4651f8f07825931c92873cb51137214a938badb7a759b85c1d95bf74f86efa \
+                    size    22301
+
+if {${name} ne ${subport}} {
+     depends_lib-append  \
+                    port:py${python.version}-tomli \
+                    port:py${python.version}-typing_extensions
```

Comment:
this dependency is only needed for Python 3.7, so that should be added only for that version 
(``` "typing-extensions; python_version<'3.8'",```)

Reply:
Thanks, done.